lib/Echidna/Transaction.hs

```diff
@@ -108,8 +108,12 @@ canShrinkTx :: Tx -> Bool
 canShrinkTx (Tx (SolCreate _) _ _ _ 0 0 (0, 0))   = False
 canShrinkTx (Tx (SolCall (_,l)) _ _ _ 0 0 (0, 0)) = any canShrinkAbiValue l
 canShrinkTx (Tx (SolCalldata _) _ _ _ 0 0 (0, 0)) = False
+canShrinkTx (Tx NoCall _ _ _ _ _ (0, 0))          = False
```

Comment:
does shrinker remove NoCalls with (0,0) delay?

Reply:
`Tx NoCall _ _ _ _ _ (0, 0)` should be removed, but it cannot be reduced more. The shrinker takes care of removing useless transactions very quickly, so this is not a problem.